build/wide-github.user.js

```diff
@@ -9,12 +9,14 @@
 // @contributor Jason Frey (https://github.com/Fryguy)
 // @contributor Marti Martz (https://github.com/Martii)
 // @contributor Paul "Joey" Clark (https://github.com/joeytwiddle)
+// @contributor Collin Chaffin (https://github.com/CollinChaffin)
 // @license     MIT; https://raw.githubusercontent.com/xthexder/wide-github/master/LICENSE
 // @version     1.2.4
 // @icon        https://raw.githubusercontent.com/xthexder/wide-github/master/icon.png
 // @homepageURL https://github.com/xthexder/wide-github
 // @supportURL  https://github.com/xthexder/wide-github/issues
 // @include     *github.com*
+// @run-at      document-end
```

Comment:
May be different in other .user.js engines... best to leave this to .user.js engine default in case naming every changes.